dotnet/src/webdriver/Command.cs

```diff
@@ -101,13 +101,29 @@ public string ParametersAsJsonString
             }
         }
 
+        /// <summary>
+        /// Serializes the parameters of the comand to JSON as UTF-8 bytes.
+        /// </summary>
+        /// <returns></returns>
+        public byte[] GetParametersAsUtf8Bytes()
```

Comment:
`Command` is an abstraction. Command to bytes... we should mention `json` in method name. Like `GetParametersAsJsoonUtf8Bytes()`